rush/rush/src/actions/RebuildAction.ts

```diff
@@ -116,6 +122,10 @@ export default class RebuildAction extends CommandLineAction {
     }
     this._rushLinkJson = JsonFile.loadJsonFile(this._rushConfiguration.rushLinkJsonFilename);
 
+    if (this._quietParameter.value) {
+      console.log(colors.yellow(`DEPRECATED: the --quiet parameter is deprecated, as builds are quiet by default`));
```

Comment:
"DEPRECATED:  The --quiet parameter will be removed soon, because builds are now quiet by default"